### ISSUE TITLE
[boss] Reliquary of Souls - Essence of Anger (BB #154)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#154]
This issue was migrated from bitbucket.
**Original Reporter:** 
**Original Date:** 19.05.2010 20:54:11 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** invalid
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/154

<hr>

I cought the RoS, there was no seeth, and than, without reason, I've lost aggro, RoS has attacked an another warrior. It was after 4-5 seconds from pull, in this time there was no chance for this warrior to build aggro using thrown weapon. Misdirect was on me, so a whole situation was increadibly wrong.

After a wipe we went to kill some trash mobs. As a tank I had a second problem, the same as on Arcatraz - i wasn't generating any aggro. I couldn't make a reload during combat so there was no way to repair a bug. Moreover, after every pull i had the same problem.

To sum up -  the RoS os bugged.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 05.12.2015 09:00:05 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/154/#comment-23823713

<hr>

I know it's been long time since this issue was reported, but I don't see it as boss bug but rather class.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 05.12.2015 09:00:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/154/#comment-23823715

<hr>
